.circleci/config.yml

```diff
@@ -87,3 +87,12 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+      - architect/push-to-app-catalog:
+          name: "package and push cert-manager-app chart"
```

Comment:
Please update both this one and the other push job to include target catalog name so the two jobs can be distinguished in CircleCI UI without looking into details.

Reply:
Great catch, I copied it from another repo, didn't know it's gonna be reflected there.